location/views.py

```diff
@@ -1,7 +1,34 @@
 from django.views import generic
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+from django.contrib.auth.decorators import login_required
+from datetime import datetime
+
+from review.models import Review
+from review.form import ReviewForm
 from .models import Location
 
 
 class LocationView(generic.DetailView):
     model = Location
     template_name = "location.html"
+
+    def get_context_data(self, **kwargs):
+        context = super(LocationView, self).get_context_data(**kwargs)
+        context["form"] = ReviewForm
```

Comment:
It's a little bit weird for me that it works with `ReviewForm` not `ReviewForm()` 🤔 
Basically `ReviewForm` is a class and I think we should pass an instance of the form.

Could you check if it works with `ReviewForm()`? Even if it works with `ReviewForm`, I think we should choose `ReviewForm()`. 

django docs also passes the instance of the form.
> form = NameForm()
https://docs.djangoproject.com/en/2.2/topics/forms/#the-view


If both things work, it's matter of readability. Because at some point, some person read this line and might come up with the same question: "Why does it work? Should it be ReviewForm()?". It's kind of misleading and possibly will lead that person to search google, which indicates the code is not clear.